src/Widget_Command.php

```diff
@@ -646,6 +643,7 @@ private function sanitize_widget_options( $id_base, $dirty_options, $old_options
 
 		// No easy way to determine expected array keys for $dirty_options
 		// because Widget API dependent on the form fields
+		// phpcs:ignore WordPress.PHP.NoSilencedErrors.Discouraged -- Whitelisting due to above reason.
```

Comment:
Whitelisting here as I was not able to generate an error, keeping the silencing operator due to that.